Makefile

```diff
@@ -140,6 +147,18 @@ docker-tools:
 
 scratch: build
 
+# ===================
+# DATABASE MIGRATIONS
+# ===================
+
+migrate-up: $(MIGRATE_BIN)
+	migrate -path sqldb/sqlc/migrations -database $(LND_DB_CONNECTIONSTRING) -verbose up
+
+migrate-down: $(MIGRATE_BIN)
+	migrate -path sqldb/sqlc/migrations -database $(LND_DB_CONNECTIONSTRING) -verbose down 1
```

Comment:
Maybe you saw this comment already, but there's a gotcha with golang-migrate: https://github.com/lightningnetwork/lnd/pull/7251#discussion_r1072411775

Reply:
This is totally on point, thanks for sharing